web/package.json

```
@@ -21,7 +21,7 @@
     "k-web-theme": "file:k-web-theme"
   },
   "bugs": {
-    "url": "https://github.com/kframework/evm-semantics/issues"
+    "url": "https://github.com/kframework/runtimeverification/issues"
```

```suggestion
    "url": "https://github.com/runtimeverification/evm-semantics/issues"
```

Is this what you meant?